src/Homework12/Author.java

```diff
@@ -0,0 +1,16 @@
+package Homework12;
+public class Author {
+    private String firstname;
+    private String surname;
+
+    public Author(String firstname, String surname) {
+        this.firstname = firstname;
+        this.surname = surname;
+    }
+    public String getFirstname() {
+        return firstname;
+    }
+    public String getSurName() {
```

Comment:
```suggestion
    public String getSurname() {
```
геттер должен называться get + название поля с большой буквы

Reply:
исправил, похоже писал как два разных слова)